sdk-config.json

```diff
@@ -1,9 +1,8 @@
 {
   "comment_sdk_config": "This is the configuration file for the React SDK",
+  "sdk_optional_configs_doc_comment": "See the document(link below) for the available optional config settings",
```

Comment:
Rather than "for the available optional config settings" would suggest "for more details on all the available config settings"